src/reducers/index.js

```diff
@@ -0,0 +1,46 @@
+import { combineReducers } from 'redux';
+import undoable from 'redux-undo';
+import background from './background';
+import columnsData from './columnsData';
+import selectedCardId from './selectedCardId';
+import cardEditing from './cardEditing';
+import { BACKGROUND_CHANGE, COLUMN_ADD, CARD_ADD, CARD_UPDATE, CARD_DELETE,
```

Comment:
I think it's not very maintainable to write a lot of named imports like this, just use `import * as something from 'something'` if imported variables don't fit in one line (applies to other files as well)